lib/mailers/debug_preview_mailer.rb

```diff
@@ -0,0 +1,10 @@
+class DebugPreviewMailer < ActionMailer::Base
+  def invalid_user(role)
+    mail(
+      from: "reply@example.com",
+      to: "missing_#{role}@example.com",
+      subject: "invalid_user_id",
+      body: "User does not exist or is not a #{role}"
+    )
+  end
+end
```

Comment:
I like the solution you came up with here! Keeps things clean and re-usable across the other mailers 🏅